lib/widgets/emoji_reaction.dart

```diff
@@ -159,12 +159,15 @@ class ReactionChip extends StatelessWidget {
                   mainAxisSize: MainAxisSize.min,
                   crossAxisAlignment: CrossAxisAlignment.center,
                   children: [
-                    // So text-emoji chips are at least as tall as square-emoji
-                    // ones (probably a good thing).
-                    SizedBox(height: _squareEmojiScalerClamped(context).scale(_squareEmojiSize)),
                     Flexible( // [Flexible] to let text emojis expand if they can
                       child: Padding(padding: const EdgeInsets.symmetric(horizontal: 3, vertical: 1),
-                        child: emoji)),
+                        child: ConstrainedBox(
+                          constraints: BoxConstraints(
+                            // minHeight so text-emoji chips are at least as tall
+                            // as square-emoji ones (probably a good thing).
+                            minHeight: _squareEmojiScalerClamped(context).scale(_squareEmojiSize)),
+                          child: Center(widthFactor: 1,
```

Comment:
Hmm ­— it feels a little awkward to have this here, because it's basically saying the same thing as the `CrossAxisAlignment.center` on the Row.